### ISSUE TITLE
Suggestion: Apply to infection prediction

### ISSUE THREAD

**Issue by EngrStudent** [SciML/DifferentialEquations.jl#788]
My kids have hand-foot-mouth and I was looking at the models.

I saw this:
https://www.nature.com/articles/s41598-020-68840-3

It is nature, middle of last year, so relatively good journal relatively recent. 

The best they had was a deep recurrent convolutional residual (homebrew) something.  
And the basic SIR models are differential equations, and they could do a job here.

I do not understand the neuralODE and the SEIR epidemiology well enough to do a decent job, but I strongly suspect that a non-shallow Neural ODE based on SEIR models might substantially outperform the homebrew substantially.

**Comment by ChrisRackauckas** [comment]
Yes, this could be an interesting project for someone to pick up.
